temperatureHumidity.py

```diff
@@ -38,25 +38,23 @@ def get_DHT11_data():
     # guarantee the timing of calls to read the sensor).
     # If this happens try again!
 
-    for i in range(1, 3):
+    tries = 2
+    for i in range(1, tries + 1):
```

Comment:
The comment in line 31 says read_retry() is being used, but read() is used. And we're only trying 2 times.